tools/test-proxy/documentation/integration-guidelines.md

```diff
@@ -0,0 +1,52 @@
+# Test-Proxy Integration Guidelines
+
+This document is intended to make help users integrate the test-proxy with their specific language. While it is not intended to get into extreme technical details, it should definitely go far enough to direct where necessary.
+
+## General Guidance
+
+Every language will need to follow the same basic process.
+
+1. Read the test-proxy readme.
+2. Spin up a local version of the test-proxy
+3. Transition a single test to flow through the test-proxy
+4. Submit a PR
+5. Add necessary additions for the test-proxy to start up in CI
+6. Update your tests to start the test-proxy automatically
+7. Verify in CI
+8. Transition the rest of your tests
+
+At any point, when you run into issues, feel free to reach out to `scbedd`. The rest of this document calls out some common features that should be supported by each language's integration with the test-proxy.
+
+## Recommendations for integrating with CI
+
+* Just like your local development experience, start with the choice of "tool" or "docker".. Choose either [test-proxy-docker.yml](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/testproxy/test-proxy-docker.yml) or [test-proxy-tool.yml](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/testproxy/test-proxy-tool.yml).
+* To enable `https` to the test-proxy, the certificate `dotnet-devcert.crt` (or `.pfx` if your language requires that) must be trusted.
+  * This logic should be implemented, then made callable in Language-Settings.ps1 function 'Import-Dev-Cert-<language>'.
+  
+## General guidance
+
+- [x] You **should** support use SSL when communicating with the test-proxy. (Local URL: `https://localhost:5001`)
+
+This can be expensive, as every language needs to trust the https certificate differently. Read [trust-cert-per-language.md](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/documentation/trusting-cert-per-language.md)
```

Comment:
```suggestion
- [x] You **should** support using SSL communication with the test-proxy. (Local URL: `https://localhost:5001`)

This can be expensive, as every language needs to trust the https certificate differently. Read [trust-cert-per-language.md](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/documentation/trusting-cert-per-language.md) for information about how each language implements this.
```